### DISCUSSION TITLE
specify lctlib.def in JSBACH?

### DISCUSSION THREAD

**Discussion by PengyangSong** [esm-tools/esm_tools#1086] [Models]
Dear ESM_tools developers and AWI-ESM2 users,

I went through JSBACH documents (https://pure.mpg.de/rest/items/item_3279802_26/component/file_3316522/content, page 210), and found the below sentence:

`The standard land cover type library file specifies 21 different PFTs, however not all PFTs might actually be used together in an experiment. For CMIP5 and CMIP6 simulations only 13 out of the 21 PFTs have been used. Below, as an example, the CMIP6 landcover library file. Data columns for PFTs that had not been used within the experiments are skipped for clearness. The hash sign (#) at the beginning of a line indicates comment lines that are ignored by the model.`

I checked on my file, I used all 21 Plant Functional Types (PFTs), and I didn't find a file for 13 PFTs.
1. Has anyone applied 13 PFTs in his runs?
2. If `lctlib.def` can be specified, would be good to add this configuration regarding CMIP-type runs in `jsbach.datasets.yaml`.

Pengyang

**Comment by PengyangSong** [comment]
My example:
Line 46 of `/work/ab0246/a270146/AWIESM-2.1-wiso/PI_CTRL_prod/run_20010101-20011231/work/lctlib.def` on **Levante**

**Comment by chrisdane** [comment]
Hi Pengyang

You can get other lctlib defs from the mpiesm tarballs available at https://code.mpimet.mpg.de/projects/mpi-esm-users/files (you need to create an account to login) and simply try them out. But I strongly recommend to discuss this with the jsbach-people.

Cheers,
Chris

**Comment by mandresm** [comment]
Thanks for the answer @chrisdane. @PengyangSong, answering to your in-person question from yesterday: I do not know who you can contact in this case. Maybe @christian-stepanek can give here a hint?

I'm transferring this issue to a discussion, as I think it's the most adequate format.
